extensions/neo4j/deployment/src/main/java/io/quarkus/neo4j/deployment/DevServicesBuildTimeConfig.java

```diff
@@ -29,4 +29,18 @@
      */
     @ConfigItem
     public Map<String, String> additionalEnv;
+
+    /**
+     * This value will be used to expose the bolt-port of the container to the outside. It must be a free port,
+     * otherwise startup will fail. The dev services will try to allocate a free port by default.
+     */
+    @ConfigItem
+    public Optional<Integer> fixedBoltPort = Optional.empty();
```

Comment:
You can use `OptionalInt` instead

Reply:
Good point, I didn't know that it was supported in the config. Changed it.